javascript/whiteboard/linked-list/README.md

```diff
@@ -19,15 +19,28 @@ Arguments: none
 Returns: a string representing all the values in the Linked List, formatted as:
 "{ a } -> { b } -> { c } -> NULL"
 
-## Whiteboard Process
+### Whiteboard Process
 
 ![linked list img](../img/linked-list.png)
 
-## Approach & Efficiency
+### Approach & Efficiency
 <!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
 I mostly followed along in class.
 
-## Solution
+### Solution
 <!-- Show how to run your code, and examples of it in action -->
 npm test linkedList
 
+<!-- _______________________________________ -->
+
+## code challenge 11
+
+for this whiteboard I am supposed to have 2 stack, 1 to enqueue from, and a 2nd to dequeue from. The goal is for the two stacks together to act as a queue would.
+
+### Whiteboard Process
+
+![pseudo-stack, cc 11](./img/pseudo-stack.png)
+
+### Approach & Efficiency
+<!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
+
```

Comment:
This goes into its _own_ readme, which you can call codeChallenge-11.md or pseudoQueue.md